src/components/UsernameEntry/actions.js

```diff
@@ -21,33 +20,18 @@ export const loadUsers = () => dispatch => {
     .catch(() => dispatch(setUsers([])));
 };
 
-const getIDByUsername = (username, users) => {
-  const currentUser = users.find(
-    user => user.name === username || user.profile.real_name === username
-  );
-  return currentUser ? currentUser.id : null;
-};
-
-export const sendSlackMessage = username => async (dispatch, getState) => {
+export const sendSlackMessage = userid => async (dispatch, getState) => {
```

Comment:
Can we make `userid` into `userID` to be consistent with what we're doing elsewhere?